libs/newsletter-workflow/src/lib/executeLaunch.ts

```diff
@@ -43,6 +36,50 @@ const getExtraValuesFromFormData = (
 	);
 };
 
+const outputToStatus = (output: {
+	success: boolean;
+}): 'REQUESTED' | 'NOT_REQUESTED' =>
+	output.success ? 'REQUESTED' : 'NOT_REQUESTED';
+
+const doNotSendEmail = () =>
+	Promise.resolve({ success: false, error: undefined });
+
+const sendOutEmailsAndUpdateStatus = async (
+	launchService: LaunchService,
+	newsletter: NewsletterData,
+) => {
+	const shouldSendTagCreationRequest =
+		!!newsletter.seriesTag && !!newsletter.composerTag;
```

Comment:
It was not clear to me that a composerTag was a requirement. I can see we need the series tag, but my understanding was that the composer / campaign tags were used for promotion?

Reply:
That makes sense - the series tag is the only one needed for the newsletter to be functional. 

We could drop the requirement for composerTag to be defined but also update the message template so it only include the 'please create the campaign tag' message if the newsletter has one.